packages/@okta/i18n/src/properties/login_cs.properties

```diff
@@ -615,10 +622,11 @@ deviceTrust.sso.expire.button = P\u0159ihlaste se pomoc\u00ED Okta Mobile
 deviceTrust.universalLink.fallback.getOktaMobile.title = Z\u00EDskejte Okta Mobile
 deviceTrust.universalLink.fallback.getOktaMobile.subtitle = Jd\u011Bte na {0}AppStore{1}, {0}vyhled\u00E1v\u00E1n\u00ED{1} pro {0}Okta Mobile{1} klikn\u011Bte na {0}GET{1} Okta Mobile. Po nainstalov\u00E1n\u00ED se p\u0159ihlaste do Okta Mobile a \u0159i\u010Fte se p\u0159\u00EDkazy pro zaji\u0161t\u011Bn\u00ED va\u0161eho p\u0159\u00EDstroje.
 
+# Device Challenge Probing and Polling
```

Comment:
this section is empty in English bundle as well as in all translated files. Not for this PR, but we shouldn't have empty sections, there is no point
@jhe-okta seems that [you added it ](https://github.com/okta/okta-signin-widget/commit/dd0d203d8b1ae20fd04ade1a9ccb9f1a66b8eb2a)to login.properties , mind removing it in some other PR?